core/trino-main/src/main/java/io/trino/operator/TaskStats.java

```diff
@@ -555,6 +556,16 @@ public Duration getFullGcTime()
         return fullGcTime;
     }
 
+    @JsonProperty
```

Comment:
Why JsonProperty ? I don't think we want to include it separately in the json representation as this is redundant info